web/src/components/Plugin/UI/limit-count.tsx

```diff
@@ -149,14 +159,14 @@ const RedisClusterForm: React.FC = () => {
               </Form.Item>
 
               <Form.Item {...FORM_ITEM_WITHOUT_LABEL}>
-                <Button
+                {fields.length < maxLength?(<Button
```

Comment:
ditto

Reply:
Thanks for your suggestion, I have modified it.